src/datadogConnector/DatadogApmConnector.ts

```diff
@@ -0,0 +1,23 @@
+import tracer from 'dd-trace'
+
+tracer.init({
+  service: process.env.DD_SERVICE || 'vagov-next-build',
+  env: process.env.DD_ENV || process.env.NODE_ENV || 'development',
```

Comment:
```suggestion
  env: process.env.DD_ENV || process.env.NODE_ENV || 'vagov-dev',
```

Just to be in line with how we label env's in DD